core/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class UnimplementedUCException(Exception):
+    """Returned when an uncaught exception is raised"""
+
+
+def unimplemented_uc_error_handler(error: UnimplementedUCException):
+    return {
+        "detail": "Uncaught workbook validation failure",
+        "status": 500,
+        "title": "Bad Request",
+    }, 500
```

Comment:
Can this be combined with `errors.py` - either file name works

Reply:
You can avoid the circular import if you avoid importing using `from`. 

Using `from core.exceptions import UnimplementedUCException` runs the entire `exceptions` module when the `failures` module is run whereas if we do `import core.exceptions as e` and then use `e.UnimplementedUCException` within the function then it lazily loads the errors module, avoiding the circular import.